### PR TITLE
Remove mention about Evented Redis [ci skip] 

### PR DESCRIPTION
**Pull request by bogdanvlviv**
Evented Redis is removed from Rails.
See #30945

### CODE REVIEW

File: guides/source/configuring.md [hunk 1]
```diff
@@ -549,7 +549,7 @@ Defaults to `'signed cookie'`.
 
 * `config.action_view.logger` accepts a logger conforming to the interface of Log4r or the default Ruby Logger class, which is then used to log information from Action View. Set to `nil` to disable logging.
 
-* `config.action_view.erb_trim_mode` gives the trim mode to be used by ERB. It defaults to `'-'`, which turns on trimming of tail spaces and newline when using `<%= -%>` or `<%= =%>`. See the [Erubis documentation](http://www.kuwata-lab.com/erubis/users-guide.06.html#topics-trimspaces) for more information.
```
Comment (rafaelfranca) [inline guides/source/configuring.md]: I kept this on purpose since erubi doesn't have documentation and the documentation is the same. Could you revert this change?

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

File: guides/source/configuring.md (hunk 1 above)
Comment (bogdanvlviv) [reply guides/source/configuring.md]: Reverted

**Review by rafaelfranca — APPROVED** [review]
